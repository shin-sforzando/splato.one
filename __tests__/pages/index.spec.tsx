import { render } from '@testing-library/react'
import Home from '@/pages'

describe('Home', () => {
  test('always should be truthy', () => {
    expect(true).toBeTruthy()
  })
  test('snapshot', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
