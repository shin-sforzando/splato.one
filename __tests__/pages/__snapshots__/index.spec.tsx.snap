// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot 1`] = `
<div>
  <span
    class="text-3xl font-bold"
  >
    splato.one
  </span>
</div>
`;
